<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QWidget" name="ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>260</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelIdFile">
        <property name="text">
         <string>ID file:</string>
        </property>
        <property name="buddy">
         <cstring>idFile</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KUrlRequester" name="idFile">
        <property name="whatsThis">
         <string>Name of ID file, as generated by mkid(1).</string>
        </property>
        <property name="filter">
         <string>ID
*|All Files</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelKeySize">
        <property name="text">
         <string>Complete tokens after:</string>
        </property>
        <property name="buddy">
         <cstring>keySize</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KIntSpinBox" name="keySize">
        <property name="whatsThis">
         <string>Minimum length of token before completions will be shown.</string>
        </property>
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="maximum">
         <number>16</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Transform Filenames</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelSrcIn">
        <property name="text">
         <string>Replace file prefix ending with key:</string>
        </property>
        <property name="buddy">
         <cstring>srcIn</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KLineEdit" name="srcIn">
        <property name="whatsThis">
         <string>Discard the first part of the file name ending with this key.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelSrcOut">
        <property name="text">
         <string>With this file prefix:</string>
        </property>
        <property name="buddy">
         <cstring>srcOut</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KLineEdit" name="srcOut">
        <property name="whatsThis">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Replacement prefix. To use the prefix of the ID file ending with the key, use %{idPrefix}. This can be used to work with remotely mounted Clearcase views, for example, set:&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;ID file:                                   /view/myview/&lt;span style=&quot; font-weight:600;&quot;&gt;topdir&lt;/span&gt;/ID &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Replace prefix ending:     /&lt;span style=&quot; font-weight:600;&quot;&gt;topdir&lt;/span&gt; &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;With prefix:                         %{idPrefix}/&lt;span style=&quot; font-weight:600;&quot;&gt;topdir&lt;/span&gt; &lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Or, to use a pre-generated ID file with a different workspace, set:&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;ID file:                                   /snapshots/&lt;span style=&quot; font-weight:600;&quot;&gt;topdir&lt;/span&gt;/ID &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Replace prefix ending:     /&lt;span style=&quot; font-weight:600;&quot;&gt;topdir&lt;/span&gt; &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;With prefix:                         /myworkspace/&lt;span style=&quot; font-weight:600;&quot;&gt;topdir&lt;/span&gt; &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Highlight Matches</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="labelUseEtags">
        <property name="text">
         <string>Highlight definitions using etags(1):</string>
        </property>
        <property name="buddy">
         <cstring>useEtags</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="useEtags">
        <property name="whatsThis">
         <string>Use etags(1) to find definitions, and highlight them.</string>
        </property>
        <property name="text">
         <string>Enabled</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelUseSuffixes">
        <property name="text">
         <string>Highlight files with suffixes:</string>
        </property>
        <property name="buddy">
         <cstring>useSuffixes</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="KLineEdit" name="useSuffixes">
        <property name="whatsThis">
         <string>Use .h;.hxx to highlight files which may contain declarations.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>idFile</tabstop>
  <tabstop>keySize</tabstop>
  <tabstop>srcIn</tabstop>
  <tabstop>srcOut</tabstop>
  <tabstop>useEtags</tabstop>
  <tabstop>useSuffixes</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
